examples/visual-tests/amp-story-player/story-loaded-amp-version.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  'story loaded and AMP version player ready': async (page) => {
+    await page.waitForTimeout(5000);
+    await page.$eval('iframe', (iframe) => {
+      const contents = iframe.contentDocument.documentElement.outerHTML;
+      iframe.src = '';
+      iframe.srcdoc = contents;
+    });
```

Comment:
Can you explain why this is needed?

Reply:
We found this is the only way Percy will render the story inside the iframe 🤷🏽‍♂️